README.md

```diff
@@ -86,10 +50,10 @@ To accept a certain payment method in your app, you only need to include that pa
 ## Sample Code
 
 ```swift
-// STEP 0: Fetch an ACCESS_TOKEN and ORDER_ID from your server.
+// STEP 0: Fetch an CLIENT_ID and ORDER_ID from your server.
```

Comment:
```suggestion
// STEP 0: Fetch a CLIENT_ID and ORDER_ID from your server.
```

Do they have to fetch a client ID? They can also have one safely hard-coded on their client as well

Reply:
In demo app, we have a merchant server call to fetch client id but they can just enter it manually as well. 
Should I say "fetch or enter"?

Reply:
I don't expect most merchants will make an API request to fetch the client ID. I think they will probably hardcode it as Sammy said.

Reply:
Addressed in 47abc2204e62fc9738a73e258d1931f9181907ea